frontend/src/Components/Endpoint/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import Button from "plaid-threads/Button";
+import Note from "plaid-threads/Note";
+
+import Table from "../Table";
+import Error from "../Error";
+import { DataItem, Categories, ErrorDataItem, Data } from "../../dataUtilities";
+
+import styles from "./index.module.scss";
+
+interface Props {
+  endpoint: string;
+  name?: string | null;
```

Comment:
Does this need to be both optional and nullable? seems like one or the other will do

Reply:
fixed